src/troubleshooting/errors.rst

```diff
@@ -0,0 +1,118 @@
+SSSD ERRORS
+##########
+
+
+SSSD process is terminated by own WATCHDOG
+******************************************
+
+.. code-block:: bash
+
+  (Fri Apr 14 15:07:19 2023) system1 sssd[sssd]: Child [1277] ('SSSDdomain':'%BE_SSSDdomain') was terminated by own WATCHDOG. Consult corresponding logs to figure out the reason.
+
+The above log indicates that the sssd_be process was blocked too long on something that was longer than ``3*10`` seconds. In order to find which operation is blocking sssd_be, enable sssd debugging by adding ``debug_level = 9`` under all sections of the ``/etc/sssd/sssd.conf`` file, especially under the ``[$domain]`` section, and wait for the issue to reoccur. Once the issue is observed, take the timestamp of the ``... was terminated by own WATCHDOG`` message and then spot the last operation before the timestamp in ``/var/log/sssd/sssd_$domain.log``. In the above example, sssd_be is getting killed, but there could be some other situation where ``sssd_nss`` or ``sssd_pam`` processes could get killed by watchdog. In this case, the troubleshooting process will be the same, but the respective sssd log needs to be checked instead of the ``/var/log/sssd/sssd_$domain.log`` file. Increasing the ``timeout`` values may server as a workaround. But its always good to spot the operation that causing the issue(sss process terminated by WD) and apply the solution accordingly. On a side note if the processing of group membership is very slow then also we can observe ``sssd_be killed by watchdog``.
```

Comment:
> Increasing the ``timeout`` values may server as a workaround. But its always good to spot the operation that causing the issue(sss process terminated by WD) and apply the solution accordingly.

I would change this to:

Increasing the sssd domain ``timeout`` values may serve as a workaround, however finding the root cause of the watchdog termination may be important.